web/test/pages/loading.test.tsx

```diff
@@ -105,18 +86,19 @@ describe("loading page", () => {
     expect(mockPush).toHaveBeenCalledWith("/tasks/some-task");
   });
 
-  it("shows modal when user has signin error and redirects user to onboarding", async () => {
+  it("saves triggers guest signin with encounteredError param when user has signin error", async () => {
```

Comment:
I'm leaving this note to come back and look at this again. Something about this test name is weird to me, but I think I need to look again.